woocommerce/google-listings-and-ads
# Add packages to allow-plugins directive and update lock file

### Changes proposed in this Pull Request:

Composer has a new [`allow-plugins`](https://getcomposer.org/doc/06-config.md#allow-plugins) setting for the `config` property to allow for specifying trusted (or untrusted) plugins from packages. This PR specifies the trusted plugins for this project.

```
automattic/jetpack-autoloader contains a Composer plugin which is currently not in your allow-plugins config. See https://getcomposer.org/allow-plugins
Do you trust "automattic/jetpack-autoloader" to execute code and wish to enable it now? (writes "allow-plugins" to composer.json) [y,n,d,?] y

dealerdirect/phpcodesniffer-composer-installer contains a Composer plugin which is currently not in your allow-plugins config. See https://getcomposer.org/allow-plugins
Do you trust "dealerdirect/phpcodesniffer-composer-installer" to execute code and wish to enable it now? (writes "allow-plugins" to composer.json) [y,n,d,?] y
```

### Detailed test instructions:

1. Run `composer install` from `develop` using Composer 2.2.0+, and observe the messages above
2. Run `composer install` from this branch and observe no messages, and the plugins are installed

### Changelog entry

> Dev – Update composer trusted plugins
